### PR TITLE
feat: desktop tutorial in template's readme

### PR DESCRIPTION
**Pull request by danil-pavlov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,188 @@
 [![official project](http://jb.gg/badges/official.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub)
 [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-# [Compose Multiplatform](https://github.com/JetBrains/compose-multiplatform) Desktop Application
+# [Compose Multiplatform](https://github.com/JetBrains/compose-multiplatform) desktop application
 
-- `./gradlew run` - run application
-- `./gradlew package` - package native distribution into `build/compose/binaries`
+> **Note**
+> We would appreciate your feedback in the public Slack channel [#compose-ios](https://kotlinlang.slack.com/archives/C0346LWVBJ4/p1678888063176359).
```
Comment (igordmn) [inline README.md]: Here should be a link to #compose-desktop

File: README.md [hunk 2]
```diff
@@ -1,6 +1,188 @@
...
+* `./gradlew run` to run the application
+* `./gradlew package` to store native distribution into `build/compose/binaries`
+
+## Next steps
+
+We encourage you to explore Compose Multiplatform further and try out more projects:
+
+* [Learn about other cases where you can use the Compose Multiplatform UI framework](https://github.com/JetBrains/compose-multiplatform#readme)
+* [Create an application targeting iOS and Android with Compose Multiplatform](https://github.com/JetBrains/compose-multiplatform-ios-android-template#readme)
+* [Complete more Compose Multiplatform tutorials](https://github.com/JetBrains/compose-multiplatform#tutorials)
```
Comment (igordmn) [inline README.md]:
```suggestion
* [Complete more Compose Multiplatform tutorials](https://github.com/JetBrains/compose-multiplatform/tutorials/README.md)
```

File: README.md [hunk 3]
```diff
@@ -1,6 +1,188 @@
...
+* [Explore some more advanced Compose Multiplatform example projects](https://github.com/JetBrains/compose-multiplatform#examples)
```
Comment (igordmn) [inline README.md]:
```suggestion
* [Explore some more advanced Compose Multiplatform example projects](https://github.com/JetBrains/compose-multiplatform/examples/README.md)
```

**Review by igordmn — COMMENTED** [review]
Thanks!

File: README.md (hunk 1 above)
Comment (igordmn) [inline README.md]:
```suggestion
> We would appreciate your feedback in the public Slack channel [#compose-desktop](https://kotlinlang.slack.com/archives/C01D6HTPATV).
```

File: README.md [hunk 4]
```diff
@@ -1,6 +1,188 @@
 [![official project](http://jb.gg/badges/official.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub)
 [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-# [Compose Multiplatform](https://github.com/JetBrains/compose-multiplatform) Desktop Application
+# [Compose Multiplatform](https://github.com/JetBrains/compose-multiplatform) desktop application
 
-- `./gradlew run` - run application
-- `./gradlew package` - package native distribution into `build/compose/binaries`
+> **Note**
```
Comment (katia-energizer) [inline README.md:5]: We don't want feedback on the desktop target. We want for iOS and web only. Let's remove this note. 
Or rewrite:
> If you have any questions about Compose for Desktop, ask them in the [#compose-desktop](https://kotlinlang.slack.com/archives/C01D6HTPATV/p1678887144652589) Slack channel. ...

File: README.md [hunk 5]
```diff
@@ -1,6 +1,188 @@
...
+### Update the plugin
+
+Before you start, ensure that you use the latest version of the Compose Multiplatform IDE support plugin:
+
+1. Check the latest release version in the [Compose Multiplatform GitHub repository](https://github.com/JetBrains/compose-multiplatform/releases) or on the [Kotlin](https://kotlinlang.org/) website.
+2. Open the `build.gradle.kts` file of your project and update the version:
+
+    ```kotlin
+    plugins {
+       kotlin("jvm") version "1.8.10"
```
Comment (katia-energizer) [inline README.md]:
```suggestion
       kotlin("jvm") version "1.8.20"
```

**Review by katia-energizer — CHANGES_REQUESTED** [review]
LGTM. Just one comment.

File: README.md [hunk 6]
```diff
@@ -1,6 +1,188 @@
...
+       id("org.jetbrains.compose") version "1.3.1"
```
Comment (igordmn) [inline README.md]:
```suggestion
       id("org.jetbrains.compose") version "1.4.0"
```

File: README.md [hunk 7]
```diff
@@ -1,6 +1,188 @@
...
+       }
+    }
+    ```
+   
+3. Create the `build.gradle.kts` file with the following content:
+
+    ``` kotlin
+    plugins {
+       kotlin("jvm") version "1.8.10"
+       id("org.jetbrains.compose") version "1.3.1"
```
Comment (igordmn) [inline README.md]:
```suggestion
       id("org.jetbrains.compose") version "1.4.0"
```

File: README.md [hunk 8]
```diff
@@ -1,6 +1,188 @@
...
+           maven("https://maven.pkg.jetbrains.space/public/p/compose/dev")
+       }
+    }
+    ```
+   
+3. Create the `build.gradle.kts` file with the following content:
+
+    ``` kotlin
+    plugins {
+       kotlin("jvm") version "1.8.10"
```
Comment (igordmn) [inline README.md]:
```suggestion
       kotlin("jvm") version "1.8.20"
```

**Pull request merged**